_includes/releases/v22.2/v22.2.0.md

```diff
@@ -0,0 +1,87 @@
+## v22.2.0
+
+Release Date: December 5, 2022
+
+With the release of CockroachDB v22.2, we've made a variety of management, performance, security, and compatibility improvements. Check out a [summary of the most significant user-facing changes](#v22-2-0-feature-summary) and then [upgrade to CockroachDB v22.2](../v22.2/upgrade-cockroach-version.html). For a release announcement with further focus on key features, see the [v22.1 blog post](LINK HERE).
+
+ADD BLURB FROM PRODUCT MARKETING
```

Comment:
@meagan-g - Can you provide copy here? And do you have a link for the v22.2 blog post?